src/pages/index.js

```diff
@@ -1,28 +1,44 @@
 import {nanoid} from 'nanoid';
+import {useState} from 'react';
 import {Helmet} from 'react-helmet';
 
 import Layout from '../components/Layout';
 import QuestionComponent from '../components/QuestionComponent';
+import QuestionForm from '../components/QuestionForm';
 
 export default function HomePage() {
-	const questionArray = [
+	const [questionArray, setQuestionArray] = useState([
 		{id: nanoid(), questionText: 'What is export default?'},
 		{id: nanoid(), questionText: 'What does clean code means?'},
 		{id: nanoid(), questionText: 'Why do we have to define keys?'},
-	];
+	]);
+
+	const createNewQuestion = event => {
+		event.preventDefault();
+		const newQuestion = {
+			id: nanoid(),
+			questionText: event.target.questionTextInput.value,
+		};
+		const newQuestionArray = [...questionArray, newQuestion];
+		setQuestionArray(newQuestionArray);
```

Comment:
```suggestion
		setQuestionArray([...questionArray, newQuestion]);
```

Redundant constant assignment. (You can keep it if it is easier for you to understand, but it is not needed here.